### ISSUE TITLE
Mermaid flowChart title can't see clearly in dark mode

### ISSUE THREAD

**Issue by evan361425** [squidfunk/mkdocs-material#7781]
### Context

Mermaid flowchart title is too dark to see in dark mode.

### Bug description

Mermaid allow write title above the flowchart

```text
---
title: Hello World
---
flowchart LR
  user --> website
```

```mermaid
---
title: Hello World
---
flowchart LR
  user --> website
```

But in dark mode, this title will be unclearly to see

![image](https://github.com/user-attachments/assets/5556111b-220d-47c2-a436-5bf11426ccae)

I tried customize CSS, but as https://github.com/squidfunk/mkdocs-material/discussions/4582 shows, this is not easy.

### Related links

- https://github.com/squidfunk/mkdocs-material/discussions/4582
- Mermaid has title theme config in [Flowchart Variables](https://mermaid.js.org/config/theming.html#flowchart-variables)
- my document with this situation: https://evan361425.github.io/essay/confidential-computing/#%E8%99%9B%E6%93%AC%E8%A8%98%E6%86%B6%E9%AB%94%E7%AE%A1%E7%90%86

### Reproduction

My palette config is

```yaml
theme:
  palette:
  - media: "(prefers-color-scheme: light)"
    scheme: default
    primary: indigo
    accent: light-blue
  - media: "(prefers-color-scheme: dark)"
    scheme: slate
    primary: blue-grey
    accent: deep orange
```

and extensions config is

```yaml
markdown_extensions:
- pymdownx.highlight
- pymdownx.superfences:
    custom_fences:
    - name: mermaid
      class: mermaid
      format: !!python/name:pymdownx.superfences.fence_code_format
```

### Steps to reproduce

1. Create mermaid codeblock same as above

```mermaid
---
title: Hello World
---
flowchart LR
  user --> website
```

2. Build the site
3. Toggle to dark mode

I'm not providing **.zip** file because it has 119 MB and I think it is not valuable for this situation, but I'll provide it if you need it!

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. First of all, please stick to [our bug reporting process](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) – you're making our life easier, and we can help you much faster by providing a mandatory [runnable minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/). For us, looking into issues that do not include runnable reproductions is pure good will, as it takes us more time to investigate.

Fixed in b368cee13f23b52335fd1ab39cacae9e26b08291. Additionally, titles of the other supported diagram types now properly render in dark mode.

**Comment by squidfunk** [comment]
Released as part of [9.5.49](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.49).
